### PR TITLE
CONSOLE-3243: Rename "master" to "control plane node" in node pages

### PR DESCRIPTION
**Pull request by rhamilto**
Replaces https://github.com/openshift/console/pull/11927 since this wasn't completed before @tvu20 finished her internship.  Thanks for the great work, @tvu20!

Note I added a custom pool named `infra` to the cluster to fully test.

Screenshot 2022-10-25 at 2 56 41 PM
Screenshot 2022-10-25 at 3 10 43 PM
Screenshot 2022-10-25 at 3 12 10 PM
Screenshot 2022-10-25 at 3 12 17 PM
Screenshot 2022-10-25 at 3 12 27 PM
Screenshot 2022-10-26 at 8 53 49 AM

### CODE REVIEW

File: frontend/packages/console-app/src/components/dashboards-page/ControlPlaneStatus.tsx
Comment (rhamilto) [inline frontend/packages/console-app/src/components/dashboards-page/ControlPlaneStatus.tsx:28]: Fixes existing incorrect casing.

File: frontend/public/components/modals/cluster-update-modal.tsx
Comment (rhamilto) [inline frontend/public/components/modals/cluster-update-modal.tsx:254]: Fixes inconsistent casing.

File: frontend/packages/console-app/src/components/nodes/NodesPage.tsx
Comment (rhamilto) [inline frontend/packages/console-app/src/components/nodes/NodesPage.tsx:515]: The role label value is `control-plane`, so make the filter reflect the actual value.

File: frontend/packages/console-app/src/components/nodes/NodesPage.tsx
Comment (rhamilto) [inline frontend/packages/console-app/src/components/nodes/NodesPage.tsx:519]: The role label value is `worker`, so make the filter reflect the actual value.

**Comment by rhamilto** [comment]
Reassigning approvers for good measure since there are a few changes from https://github.com/openshift/console/pull/11927.

QE Approver:
/assign @yapei

Docs Approver:
/assign @opayne1

PX Approver:
/assign @RickJWagner

Console Approver:
/assign @jhadvig

File: frontend/public/components/modals/cluster-update-modal.tsx
Comment (rhamilto) [inline frontend/public/components/modals/cluster-update-modal.tsx:254]: Make casing consistent and fix missing pluralization.

**Comment by rhamilto** [comment]
cc: @megan-hall

**Review by jhadvig — APPROVED** [review]
@rhamilto thanks for finishing this work! Looks good 👍 
Also kudos to @tvu20 for the work!

/lgtm

**Comment by rhamilto** [comment]
@yapei, can you please review?

**Comment by rhamilto** [comment]
/assign @XiyunZhao

@XiyunZhao, https://issues.redhat.com/browse/CONSOLE-3243 lists you as the QE contact.  Can you please verify?

**Comment by XiyunZhao** [comment]
@rhamilto Sure, working on it, will give feed back before today OOF

**Comment by XiyunZhao** [comment]
Hi @rhamilto, I have some questions about the changes 
- [ ] We use the node-role.kubernetes.io/control-plane role instead of node-role.kubernetes.io/master
- [ ] We only use master when displaying a name of a resource
**1. For the summary by Sam, do we need to care about the label of node-role.kubernetes.io/master in the YAML file for the nodes? I am confused about this part, as the roles column in the Nodes pages seems no need to change**
    Before changes are on the left. 
image
image
**2. the translation is missing** 
image

**Comment by rhamilto** [comment]
> Hi @rhamilto, I have some questions about the changes

Hi, @XiyunZhao!

> * [ ]  We use the node-role.kubernetes.io/control-plane role instead of node-role.kubernetes.io/master

That is correct.  But `node-role.kubernetes.io/master` has *not* been removed from the resource.

> * [ ]  We only use master when displaying a name of a resource
>   **1. For the summary by Sam, do we need to care about the label of node-role.kubernetes.io/master in the YAML file for the nodes? I am confused about this part, as the roles column in the Nodes pages seems no need to change**
>   Before changes are on the left.
>   image
>   image

That is correct as the label has not been removed from the resource.  We want to display any/all labels that appear on the resource.  If/when the `master` label is removed from the resource, it will no longer appear in the UI.

>   **2. the translation is missing**
>   image

That's because the filter label is now using the same English-only string from the label since the label isn't translated in the table row.  My assumption is we don't want to translate the literal values.

**Comment by XiyunZhao** [comment]
@rhamilto Sure, thanks for your quick and helpful reply. As the text is already in i18n format, so I will do further checking after this PR is merged.

This PR has been tested on the cluster start by cluster-bot (including single node cluster). The text of "Master" has been updated to "control-plan"(on Overview, MCP, cluster setting page and etc.), and no issue was found during the cluster update.
/label qe-approved

**Pull request merged**
